### DISCUSSION TITLE
Does tailwind JIT process custom plugins?

### DISCUSSION THREAD

**Discussion by logan-jobzmall** [tailwindlabs/tailwindcss#5076] [Help]
It seems as though when I enable JIT, my custom plugins just don't run at all. Thanks for the information

**Comment by simonswiss** [comment]
**EDIT**: sorry, I completely missed the word `plugin` here 😅 - assuming you have the `purge` array filled, are you able to share a minimal code repro of your situation?

But to answer your question – JIT **does** support custom plugins. Here's a very contrived example adding two utility classes via the plugin API:

https://play.tailwindcss.com/NLGt3lmi4A

---

Hey!

Definitely missing a lot of context here, but my guess is you haven't specified the files to "scan" for the JIT engine, in your `purge` option in the config.

_Nothing_ being generated is a classic case of not having specified which files to watch:

![image](https://user-images.githubusercontent.com/485747/127803607-657e93e0-3f80-48d5-968f-9804fa5707d5.png)

https://tailwindcss.com/docs/just-in-time-mode#enabling-jit-mode

If that's not the case, please share some code and more details about your specific situation 👍
